app/controllers/v0/sessions_controller.rb

```diff
@@ -157,6 +156,18 @@ def handle_login_error
       end
     end
 
+    def log_persisted_session_and_warnings
+      obscure_token = Session.obscure_token(@session.token)
+      Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
+      # We want to log when SSNs do not match between MVI and SAML Identity. And might take future
+      # action if this appears to be happening frquently.
+      if @current_user.loa3? && @current_user&.identity&.ssn && @current_user&.va_profile&.ssn
```

Comment:
Maybe this could be a method on user or decompose the conditional?
```ruby
def mismatched_ssns?
  return false unless @current_user.loa3? && @current_user&.identity&.ssn && @current_user&.va_profile&.ssn
  @current_user.identity.ssn != @current_user.va_profile.ssn
end
```

Reply:
made this change.